### ISSUE TITLE
[Feature Request] Improve usability in Shortcut mapper: Faster mouse wheel scrolling

### ISSUE THREAD

**Issue by rillig** [notepad-plus-plus/notepad-plus-plus#5611]
Extracted from #2563:

## Faster mouse wheel scrolling

When scrolling via the mouse wheel, each click corresponds to a change of one line. Almost all other programs scroll 3 lines per click. Scrolling only a single line makes Notepad++ feel slow.

When scrolling with the mouse wheel, each click should scroll 3 lines.

**Comment by MetaChuh** [comment]
@rillig 

many thanks for your feature request.

the amount of lines scrolled in notepad++ is defined by the windows mouse settings.
please go to your windows `control panel > mouse > wheel` and select the number of lines you want to scroll with one mouse wheel tick.

if you have an optional mouse driver, please check if it overrides your windows settings.

thank you for your help and best regards.

**Comment by eljefe7000** [comment]
@MetaChuh 

Sorry to comment on a closed issue, but in my testing, it appears that the control panel setting has no effect on the scroll rate for the shortcut mapper in NP++.  Before the request was closed, I found a couple lines of code where the scroll rate is hardcoded to 1 line (one line of code for going up and one for going down).  When I changed it to a larger number, I could see the scroll rate change.  If we can reopen this request, then I can set it to a larger number.
